internal/util-collection/src/main/scala/sbt/INode.scala

```diff
@@ -35,6 +34,7 @@ abstract class EvaluateSettings[Scope] {
         case None    => constant(() => o.f(None))
         case Some(i) => single[s, T](transform(i), x => o.f(Some(x)))
       }
+      case x if x == StaticScopes => strictConstant(allScopes.asInstanceOf[T]) // can't convince scalac that StaticScopes => T == Set[Scope]
```

Comment:
`StaticScopes` directly extends `Initialize[Set[Scope]]` so the ordering shouldn't matter here.